src/client/cli/cmd/launch.cpp

```diff
@@ -435,6 +436,23 @@ mp::ReturnCode cmd::Launch::request_launch(const ArgParser* parser)
         if (timer)
             timer->pause();
 
+        std::vector<std::string> warning_aliases;
+        auto n = reply.aliases_to_be_created_size();
+        for (auto i = 0; i < n; ++i)
```

Comment:
Is it not possible to just iterate over `reply.aliases_to_be_created()` directly like:

`for (const auto& alias_to_be_created : reply.aliases_to_be_created()`

or similar?

Reply:
Oh, I didn't know RPC messages could be iterated like that. I changed it, thanks!